vscode-extension/package.json

```diff
@@ -137,10 +137,8 @@
     "arrowParens": "avoid",
     "bracketSameLine": true,
     "bracketSpacing": false,
-    "requirePragma": true,
```

Comment:
I don't think we want this here, it seems tedious and unnecessary.

Reply:
Given that this is TS, I'd agree.